### ISSUE TITLE
Integral limits under and above integral sign

### ISSUE THREAD

**Issue by OvidiusCicero** [typst/typst#1522]
### Description

I would like to have a global way to set integral limits under and below the integral sign as opposed to left top and left bottom—an equivalent of the `intlimits` option of `amsmath` latex package

It is possible to redefine the integral command by `#let integral = math.limits(math.integral)` but it is not possible to redefine the actual ∫ sign because it cannot be an identifier. [yes I like to use as many Unicode characters directly as possible).

`#show "∫": math.limits(math.integral)` does not have an effect at all.

Basically it would be nice to have same knobs for `sym.integral` as we have `sym.sum` which can be manipulated globally.

### Use Case

This is a convention which used to be widespread in Germany and France—I believe. For `integral_(very long and/or verbose limits` it often looks better.

**Comment by Enivex** [comment]
This seems to work:

```
#show math.attach : it => {
  if it.base == [∫]{
    let temp = it.fields()
    math.attach(math.limits(temp.remove("base")),..temp)
  } else{
    it
  }
}
```

**Comment by sitandr** [comment]
Dublicate of #1443.

I tried to fix it in #1452, but, unfortunately, that PR is still pending. Probably, there would be some better way to fix it rather than my in some time.

**Comment by Enivex** [comment]
Yes but like @sitandr says it should probably be limited to block math.
